recipes/gtk-doc-stub/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class GtkDocStubConan(ConanFile):
+    name = "gtk-doc-stub"
+    homepage = "https://gitlab.gnome.org/GNOME/gtk-doc-stub"
+    description = "Helper scripts for generating GTK documentation"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPLv2-or-later"
```

Comment:
```suggestion
    license = "GPL-2.0-or-later"
```

GPLv2 is valid, but let's try to use the same short name too keep a pattern.

https://spdx.org/licenses/GPL-2.0-or-later.html